### ISSUE TITLE
Allow users to choose 3+ languages in "Show translations in:".

### ISSUE THREAD

**Issue by RyckRichards** [Tatoeba/tatoeba2#2617]
**Story**
Sometimes I want to study a language and I read a sentence translated into ... a language. However, sometimes it's not translated into language A. Then I have to make another research and on "Show translations in:", I have to choose another language

*Example:*
I want to study German. So I look for sentences translated in Portuguese. Sometimes a sentence is not translated into that language and I have to

1) Click on "Show translations in:"
2) Choose another language I can read in
3) Keep repeating the process until I find a translation

**Idea**
Allow users to choose 3+ languages in "Show translations in:".

**Comment by jiru** [comment]
How about allowing to select "all profiles languages" in "Show translations in:"?

**Comment by RyckRichards** [comment]
That would be great too.

**Comment by alanfgh** [comment]
I agree that "All profile languages" (note: no "s" after "profile") would be good.

**Comment by jiru** [comment]
Related: #2562

How about solving this problem using the existing expand/collapse translations system? When selecting "Show translations in: all languages" (or "To: any language" in search), the results would show translations in profile languages only, along with a "Show n more translations" button. The button, instead of just showing *some more translations*, would show *translations in all the other languages that are not in my profile*. We could even set this pattern as default pretty much everywhere, not only in pages having a "Show translations in:" filter.

Maybe such behavior should only get activated when you have two or more profile languages. Otherwise we stick to the current "hide translations if there are more than 5 of them".
